asserts/database.go

```diff
@@ -148,10 +148,23 @@ func OpenDatabase(cfg *DatabaseConfig) (*Database, error) {
 
 	trustedBackstore := NewMemoryBackstore()
 
-	for _, accKey := range cfg.TrustedKeys {
-		err := trustedBackstore.Put(AccountKeyType, accKey)
-		if err != nil {
-			return nil, fmt.Errorf("error loading for use trusted account key %q for %q: %v", accKey.PublicKeyID(), accKey.AuthorityID(), err)
+	for _, a := range cfg.Trusted {
+		switch a.Type() {
```

Comment:
Type-specific case blocks are there for the errors (I forgot there's no concept of a generic "primary key path" in asserts).